docs/tutorials/index.md

```diff
@@ -1,8 +1,43 @@
 # Introduction
 
+*Paragraph providing brief overview of this module*
+
+## Learning Objectives
+
+After completing this module, tutorial leads will:
+
+* create hackweek [learning objectives](https://www.sciencedirect.com/topics/social-sciences/learning-objective) using Bloom's Taxonomy
```

Comment:
Doesn't need to happen in this PR, but given the more formal content, we can add a glossary page which I think is nice for things like “learning outcome”, “Short-Format Training”, “Bloom’s Taxonomy” (http://book-template.hackweek.io/reference/glossary.html)